README.md

```diff
@@ -20,6 +20,7 @@ Usage:
 Flags:
       --anonymize                    flag to anonymize collected NSX resources (default false)
       --color                        flag to enable color output (default false)
+      --disjoint-hint stringArray    comma separated list of groups/tags that are VMs disjoint, needed for an effective and sound synthesis process, can specify more than one hint (example: "--disjoint-hint vm1,vm2 --disjoint-hint vm3,vm4")
```

Comment:
```suggestion
      --disjoint-hint stringArray    comma separated list of groups/tags that are VMs disjoint, needed for an effective and sound synthesis process, can specify more than one hint (example: "--disjoint-hint frontend,backend--disjoint-hint app,web")
```
is it possible to specify `-disjoint-hint app,web,db` ?

Reply:
I dont follow, why not?

Reply:
fixed

Reply:
> I dont follow, why not?

this was a question, not a statement.
if possible, please update the example to demonstrate with 3 values (one could think from the example that it's only for pairs).